### PR TITLE
Update branch and labels

### PR DESCRIPTION
**Pull request by lulf**
* Use consistent ordering of fields
* Consistent use of newlines

### CODE REVIEW

File: address-space-controller/image-template.yaml [hunk 1]
```diff
@@ -50,5 +51,5 @@ run:
           - "/opt/address-space-controller.jar"
 osbs:
       repository:
-            name: containers/amqmaas-1
-            branch: rh-amqmaas-1.0-address-space-controller-openshift-rhel-7
+            name: containers/amq-online-1
+            branch: amq7-amq-online-1-address-space-controller-openshift-container
```
Comment (vbusch) [inline address-space-controller/image-template.yaml]: @lulf the branch will need to match what is being entered into comet and should have rhel-7

File: auth-plugin/image-template.yaml [hunk 2]
```diff
@@ -6,13 +6,14 @@ description: "AMQ Online SSO for authentication"
 from: jboss/openjdk18-rhel7:1.1
 labels:
     - name: "com.redhat.component"
-      value: "rh-amqmaas-1-amqmaas10-auth-plugin-openshift-container"
+      value: "amq7-amq-onlie-1-auth-plugin-openshift-container"
```
Comment (vbusch) [inline auth-plugin/image-template.yaml]: sp: onlie (online)

File: auth-plugin/image-template.yaml [hunk 3]
```diff
@@ -49,7 +52,8 @@ run:
       user: 185
       entrypoint:
           - "/keycloak-plugin/bin/init-keycloak.sh"
+
 osbs:
       repository:
-            name: containers/amqmaas-1
-            branch: rh-amqmaas-1.0-auth-plugin-openshift-rhel-7
+            name: containers/amq-online-1
+            branch: amq7-amq-onlie-1-auth-plugin-rhel-7
```
Comment (vbusch) [inline auth-plugin/image-template.yaml]: sp: onlie (online)

**Review by vbusch — CHANGES_REQUESTED** [review]
LGTM except for the 2 typos (onlie)

**Comment by lulf** [comment]
@vbusch updated

**Review by vbusch — APPROVED** [review]

**Pull request merged**